hack/local-up-cluster.sh

```diff
@@ -688,111 +687,113 @@ function start_kubelet {
 
     priv_arg=""
     if [[ -n "${ALLOW_PRIVILEGED}" ]]; then
-      priv_arg="--allow-privileged=${ALLOW_PRIVILEGED} "
+      priv_arg="--allow-privileged=${ALLOW_PRIVILEGED}"
     fi
 
-    cloud_config_arg="--cloud-provider=${CLOUD_PROVIDER} --cloud-config=${CLOUD_CONFIG}"
+    cloud_config_arg=("--cloud-provider=${CLOUD_PROVIDER}" "--cloud-config=${CLOUD_CONFIG}")
     if [[ "${EXTERNAL_CLOUD_PROVIDER:-}" == "true" ]]; then
-       cloud_config_arg="--cloud-provider=external"
-       cloud_config_arg+=" --provider-id=$(hostname)"
+       cloud_config_arg=("--cloud-provider=external")
+       cloud_config_arg+=("--provider-id=$(hostname)")
```

Comment:
To make the following fixes work correctly:

In ./hack/local-up-cluster.sh line 759:
      ${cloud_config_arg}
      ^-----------------^ SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.